tests/scripts/all.sh

```diff
@@ -1711,20 +1711,32 @@ component_test_crypto_full_md_light_only () {
     make test
 }
 
-component_test_full_no_cipher_no_psa_crypto () {
+component_test_full_no_cipher_with_legacy () {
```

Comment:
In #9185 I just remove this component as it is not relevant anymore in 4.0 context.